pyproject.toml

```diff
@@ -18,6 +18,8 @@ ignore = ["nbgrader/labextension/**", "yarn.lock"]
 
 [tool.jupyter-releaser.hooks]
 after-bump-version = ["python tools/post-bump.py"]
+before-build-npm = ["jlpm"]
```

Comment:
Wondering if we still need the `python -m pip install jupyterlab~=3.1` as well.

Reply:
I removed it as `jupyterlab` was in `build-system` requires.

But after having a look at `jupyter_releaser` it seems that the npm is built before the python : 
https://github.com/jupyter-server/jupyter_releaser/blob/023124cbb7c7685205c39d653c2d90d6563481fb/jupyter_releaser/actions/draft_release.py#L68-L72.
So that requirement will not be available. 
Thanks @jtpio